tests/pipelines/test_pipelines_common.py

```diff
@@ -795,6 +800,135 @@ def test_cfg(self):
         assert out_cfg.shape == out_no_cfg.shape
 
 
+@is_staging_test
+class PipelinePushToHubTester(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls._token = TOKEN
```

Comment:
You can use `token=cls._token` as arg everywhere if you prefer but honestly passing `token=TOKEN` is fine as well and is more explicit IMO